style.css

```diff
@@ -405,8 +405,27 @@ ul {
   color: #6070ff;
 }
 
-.fotter-contact-form-text .btn-footer-container:hover {
+.fotter-contact-form-text .btn-footer-container:active {
   cursor: pointer;
```

Comment:
####  You have done great integrating the form with the [Formspree](https://formspree.io/). But The other requirement that the previous reviewer mentions are not been met. This includes... 
- [ ] 1. There appears to be an unnecessary extra space at the bottom that creates an awkward gap and makes the section look unbalanced.
- [ ] 2. The button interactions, such as, active (when pressed), disabled, and enable (default state) are not styled as shown in the Figma template. Please make the proper adjustments to achieve a better UI. Remember, as a frontend engineer we are expected to deliver a clone of the original design. 
![image](https://github.com/malodev1234/Portfolio_setup_mobile_first/assets/69077061/6754df3d-594b-45a8-ba9f-a288524c4073)

Reply:
hey i just set this problem ..try just to check the deployment you will see that everything is good as you asked for please

Reply:
plz visit this link https://malodev1234.github.io/Portfolio_setup_mobile_first/ you will see by yourself